tests/utils/test_name_validation.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from app.utils.validation_utils import is_name_valid
+
+class TestNameValidation(unittest.TestCase):
+    def test_empty_name(self):
+        name = ""
+        is_valid = is_name_valid(name)
+        self.assertFalse(is_valid)
+
+    def test_valid_name(self):
+        name = "valid"
+        is_valid = is_name_valid(name)
+        self.assertTrue(is_valid)
+
+    def test_invalid_name_with_digits(self):
+        name = "invalid123"
+        is_valid = is_name_valid(name)
+        self.assertFalse(is_valid)
+
+    def test_invalid_name_with_digits_and_special_characters(self):
+        name = "1234@"
+        is_valid = is_name_valid(name)
+        self.assertFalse(is_valid)
+
```

Comment:
Can you add a test, for Names with spaces, for example "FirstName SecondName", because we have to allow spaces between words.

Reply:
Sure!